File: Placila_storitve/models/payment.js


class Payment {
    constructor(PaymentID, UserID, EventID, TicketID, Amount, PaymentMethod, TransactionDate, 
        Status, PaymentToken, CardLastFour, TransactionID, RefundableUntil) {
        this.PaymentID = PaymentID;
        this.UserID = UserID;
        this.EventID = EventID;
        this.TicketID = TicketID;
        this.Amount = Amount;
        this.PaymentMethod = PaymentMethod;
        this.TransactionDate = TransactionDate;
        this.Status = Status;
        this.PaymentToken = PaymentToken;
        this.CardLastFour = CardLastFour;
        this.TransactionID = TransactionID;
        this.RefundableUntil = RefundableUntil;
    }
}

module.exports = Payment;

const paymentRoutes = require('./routes/payments');
app.use('/payments', paymentRoutes);

File: Vstopnice_storitve/SUA_vstopnice/SUA_vstopnice/Models/Payment.cs
using MongoDB.Bson.Serialization.Attributes;
using MongoDB.Bson;

namespace SUA_vstopnice.Models
{
    public class Payment
    {
        
        public string PaymentID { get; set; }
        public string UserID { get; set; }
        public string EventID { get; set; }
        public string TicketID { get; set; }
        public decimal Amount { get; set; }
        public string Currency { get; set; }
        public string PaymentMethod { get; set; }
        public DateTime TransactionDate { get; set; }
        public string Status { get; set; }

        public string PaymentToken{ get; set; }
        public int CardLastFour { get; set; }
        public string TransactionID { get; set; }
        public DateTime RefundableUntil { get; set; }
    }
}
